README.md

```diff
@@ -103,6 +103,11 @@ So many conferences got this wrong, and it's so simple. It shouldn't be anything
 * Consider having varying talk lengths. Having only 30 minute talks can feel monotonous - throw in some 15's or 20's to keep the rhythm dynamic.
 * Have many, long breaks. While the talks are interesting, the real magic at conferences happens in the conversations with people who are as passionate about the topic as you are. ReactRally is one of the conferences who does this really well with a 30 minute break after every 2 talks.
 * Schedule a big-name speaker to the final block of the day so people are more likely to stay for the whole day - if you're a new or less-known name, it sucks to speak at the end of the day to a half-empty audience.
+* Keep a timer somewhere in the front row where the speaker can clearly see it.
+* **Do not**, under any circumstances, interrupt the speaker while he's still talking. Allow your schedule to be a bit flexible. If you're 10 minutes behind, take 5 minutes from the next break or from the lunch break, but let the speakers finish their talks. It's all about respecting the work they put to be on stage.
```

Comment:
I'm unsure about this one. The speaker overrunning is a huge sign of disrespect to the organisers and audience. I feel that while a certain buffer is acceptable, going 10 minutes over time on what was supposed to be a 30 minute talk is not.

Reply:
If we're going to make this change, please make it gender neutral language. Do not assume "he's" talking.

Reply:
thanks! I missed that, did not had a last check before the commit